src/lib/components/table/terra-table-data-source.ts

```diff
@@ -102,6 +102,11 @@ export abstract class TerraTableDataSource<D, F = unknown> extends DataSource<D>
     }
     private _paginator: MatPaginator | undefined;
 
+    /** A stream that emits whenever data has been requested from the server. */
+    // tslint:disable-next-line:member-ordering
+    public searched$: Observable<boolean>;
+    private _searched$: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Comment:
Nur ein Vorschlag zu einem alternativen Namen:
```suggestion
    public dataRequested$: Observable<boolean>;
    private _dataRequested$: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Reply:
oder vllt auch dataReceived